zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/authorization/DbRoleState.java

```diff
@@ -55,4 +56,28 @@ public DbRoleState(
         zeebeDb.createColumnFamily(
             ZbColumnFamilies.ROLE_BY_NAME, transactionContext, roleName, fkRoleKey);
   }
+
+  @Override
+  public void createRole(final RoleRecord roleRecord) {
+    roleKey.wrapLong(roleRecord.getRoleKey());
+    persistedRole.setRole(roleRecord);
+    roleColumnFamily.insert(roleKey, persistedRole);
+
+    roleName.wrapString(roleRecord.getName());
+    roleByNameColumnFamily.insert(roleName, fkRoleKey);
+  }
+
+  @Override
+  public RoleRecord getRole(final long roleKey) {
```

Comment:
🔧 WDYT about returning an Optional instead of null?

Reply:
👍